.github/workflows/pytest.yml

```diff
@@ -27,53 +27,56 @@ jobs:
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version  }}
+      - name: Display python version
+        run: python -c "import sys; print(sys.version)"
 
-    - name: Install pytest
-      run: pip install pytest
-      
-    # # Get cache
-    # - name: Cache pip
-    #   uses: actions/cache@v2
-    #   with:
-    #     # This path is specific to Ubuntu
-    #     path: ~/.cache/pip
-    #     # Look to see if there is a cache hit for the corresponding requirements file
-    #     key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
-    #     restore-keys: |
-    #       ${{ runner.os }}-pip-
-    #       ${{ runner.os }}-
+      # Install pip and packages
+      - name: Install pip
+        run: python -m pip install --upgrade pip
 
-    # Install clang and it's python inteface via apt
-    - name: Add llvm keys
-      run: |
-        wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-        echo 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
-        echo 'deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
-    - name: Install libclang and its python bindings
-      run: sudo apt-get update -y && sudo apt-get install -y libclang-11-dev python3-clang-11
+      - name: Install pytest
+        run: pip install pytest
 
-    # Add dist-package to path to enable apt installed python3-clang import
-    - name: Add dist-packages to PYTHONPATH
-      run: echo "::set-env name=PYTHONPATH::${PYTHON_PATH}:/usr/lib/python3/dist-packages"
-    - name: Display PYTHONPATH
-      run: python -c "import sys; print('\n'.join(sys.path))"
-      
-    # Test with pytest
-    - name: Test with pytest
-      run: |
-        cd bindings/python
-        pytest --verbose
+      # # Get cache
+      # - name: Cache pip
+      #   uses: actions/cache@v2
+      #   with:
+      #     # This path is specific to Ubuntu
+      #     path: ~/.cache/pip
+      #     # Look to see if there is a cache hit for the corresponding requirements file
+      #     key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+      #     restore-keys: |
+      #       ${{ runner.os }}-pip-
+      #       ${{ runner.os }}-
+
+      # Install clang and it's python inteface via apt
+      - name: Add llvm keys
+        run: |
+          wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          echo 'deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
+          echo 'deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic main' | sudo tee -a /etc/apt/sources.list
+      - name: Install libclang and its python bindings
+        run: |
+          sudo apt-get update
+          sudo apt-get cache
+          sudo apt-get install -y libclang-11-dev python3-clang-11
```

Comment:
This being the only diff if ignoring whitespace changes, I'm prone to ask:
- why remove `-y` from update?
- what is `apt-get cache`? On my machine:
```
$ apt-get cache
E: Invalid operation cache
```

Reply:
@kunaltyagi @aponza Please ignore this PR, it's just for re-running and trying out some stuff for fixing broken CI.

> why remove -y from update?

It doesn't require it.

> what is apt-get cache

A mistake :joy: